src/proxy.js

```diff
@@ -258,7 +259,7 @@ const mapTraps = makeTrapsForGetters({
 	keys: state => () => latest(state).keys(),
 	values: iterateMapValues,
 	entries: iterateMapValues,
-	[Symbol.iterator]: iterateMapValues
+	[hasSymbol && Symbol.iterator]: iterateMapValues
```

Comment:
I think `hasSymbol ? Symbol.iterator : "@@iterator"` would be better; it avoids creating "null" keys, and I've seen in other libs that `@@standardSymbolName` is a common fallback for the official symbols

Reply:
I agree, it looks cleaner. But if env doesn't have Symbol, these code won't fire. Or Did you mean only readability?